manifests/s/SMLNJ/SMLNJ/Legacy/110.99.4/SMLNJ.SMLNJ.Legacy.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SMLNJ.SMLNJ.Legacy
+PackageVersion: 110.99.4
+PackageLocale: en-US
+Publisher: Fellowship of SML/NJ
+PublisherUrl: https://github.com/smlnj
+PublisherSupportUrl: https://github.com/smlnj/legacy/issues
+PackageName: Standard ML of New Jersey
+PackageUrl: https://www.smlnj.org/
+License: BSD-3-Clause License
+LicenseUrl: https://github.com/smlnj/legacy/raw/main/LICENSE
+Copyright: Copyright © 2004-2021, SML/NJ Fellowship.
+ShortDescription: |-
+  This project is the old version of Standard ML of New Jersey that continues
+  to support older systems (e.g., 32-bit machines).
+Tags:
+- functional-programming
+- sml
+- standard-ml
+ReleaseNotes: |-
+  This is a bug-fix release of the development version of SML/NJ. The most
+  important fix is that the system now correctly supports Linux and the latest
+  versions of macOS. This release also incorporates a number of fixes from the
+  legacy version 110.99.4.
```

```suggestion
  This version is primarily a bugfix release, but it does include a small number of enhancements 
  as well as support for macOS 14 (Sonoma).
```

From https://www.smlnj.org/index.html. The ReleaseNotes you added seem to be for version `2023.1`